include/natalie/array_packer/packer.hpp

```diff
@@ -128,9 +128,33 @@ namespace ArrayPacker {
                     });
                     break;
                 }
-                case 'x':
-                    // TODO
+                case 'x': {
+                    // Asterisks have no effect on this directive.
+                    if (token.star)
+                        break;
```

Comment:
I don't understand how `break` can work here. That means the 'x' token is ignored? But in the Ruby docs [it says](https://github.com/seven1m/ruby_spec/blob/6058ec330b3ed47c55c1dd6feef3ff8c33d6a0bb/core/string/unpack/x_spec.rb#L51-L53):

>it "moves the read index to the end of the string when passed the '*' modifier"